tests/interface/testCmdTlmServer.cpp

```diff
@@ -18,6 +18,19 @@ typedef struct _serverData {
     struct sockaddr_in serverAddr;
 } serverData_t;
 
+// Data structure of the test telemetry, which has a bigger size
```

Comment:
Please say what struct this is larger than.

It appears perhaps you just have two structs you send in this test: a slightly larger struct and a slightly smaller struct. But I don't know if you are also sending command ack, which is larger than either of these, and/or configuration (about which I have no idea).

Reply:
This is just to verify the condition to use the same code for the different message size (in the controller, we will have the `telemetry` and `config` to use the same message queue and allocated memory).

Reply:
Add more document in the code in commit: 7473889. Thanks!